sdk/storage/storage-blob/src/Clients.ts

```diff
@@ -2451,9 +2451,7 @@ export class BlobClient extends StorageClient {
       //   doing the same in the SDK side so that the user doesn't have to replace "\" instances in the blobName
       blobName = blobName.replace(/\\/g, "/");
 
-      if (!blobName) {
-        throw new Error("Provided blobName is invalid.");
```

Comment:
Mention in the changelog

Reply:
Also, I recommend running the live tests once before merging.

Reply:
Added changelog. And I think running CI is enough as there is no service side change.
Anyway, will run the live tests.